shipyard_blueprints/trello/shipyard_trello/__init__.py

```diff
@@ -1 +1 @@
-from .trello import TrelloClient
\ No newline at end of file
+from trello import TrelloClient
```

Comment:
what is interesting is this change was done by my ide and the cli test commands i done still worked